tests/unit/test_main.py

```diff
@@ -246,5 +248,69 @@ def test_preference_args(self):
         prefs = main.preference(None, prefs_args)
         self.assertEquals(len(prefs), 0)
 
+
+class TestCheckMozregresionVersion(unittest.TestCase):
+    @patch('requests.get')
+    @patch('sys.stdout')
+    def test_pypi_is_accesible(self, stdout, get):
+        options = None
+        logger = Mock('mozlog.structured.commandline.setup_logging')(
```

Comment:
I think that just looger = Mock() is sufficient here. Plus, you won't have to mock stdout.